Jenkinsfile

```diff
@@ -33,6 +33,8 @@ pipeline {
         stage('clear deployment') {
             steps {
                 sh 'docker image prune -a -f'
+                sh 'podman image prune -a '
+                sh 'make clean-onprem'
```

Comment:
maybe it should be a different order? 
1. clear-onprem
2. clear-deployment
3. docker image prune
4. podman image prune